lib/patriot/worker/servlet/worker_api_servlet.rb

```diff
@@ -0,0 +1,65 @@
+require 'sinatra/base'
+require 'sinatra/contrib'
+
+module Patriot
+  module Worker
+    module Servlet
+      # excepton thrown when a woker is not accessible
+      class WorkerInaccessibleException < Exception; end
+      # provide worker management functionalities
+      class WorkerAPIServlet < Patriot::Worker::Servlet::APIServletBase
+        register Sinatra::Contrib
+
+        HOST_KEY = "host"
+        VERSION_KEY = "version"
+        CLASS_KEY = "class"
+        STARTED_AT_KEY = "started_at"
+        CONFIG_KEY = "config"
+
+        STATE_KEY = "state"
+
+        LOCALHOST_EXPS = ["localhost", "127.0.0.1"]
+
+        set :show_exceptions, :after_handler
+
+        get '/' do
+          worker_hosts = @@config.get(Patriot::Util::Config::WORKER_HOST_KEY).map do |h|
```

Comment:
`config.get` seems to return a string when not-comma-separated string is given, then raises an error here.
In case such a case happens